<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Логическое сложение/умножение файлов</string>
  </property>
  <property name="windowIcon">
   <iconset resource="q.qrc">
    <normaloff>:/icon.ico</normaloff>:/icon.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="outputLBL">
       <property name="text">
        <string>Выберите выходной файл:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="outputLED">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="outputPBN">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="q.qrc">
         <normaloff>:/folder.png</normaloff>:/folder.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="inputLBL">
       <property name="text">
        <string>Выберите исходные файлы:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="inputPBN">
       <property name="maximumSize">
        <size>
         <width>33</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="q.qrc">
         <normaloff>:/folder.png</normaloff>:/folder.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="inputPTE">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="andRB">
       <property name="text">
        <string>Логическое 'И'</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="orRB">
       <property name="text">
        <string>Логическое 'ИЛИ'</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="executePBR">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="executePBN">
     <property name="text">
      <string>Выполнить</string>
     </property>
     <property name="icon">
      <iconset resource="q.qrc">
       <normaloff>:/control.png</normaloff>:/control.png</iconset>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="q.qrc"/>
 </resources>
 <connections/>
</ui>
